src/gui/Layout.h

```diff
@@ -39,6 +39,7 @@ class Layout
 	/**
 	 * Adjusts the layout size to the given values
 	 */
+	// This should never happen from the outside, since it must recalculated for every change
```

Comment:
The method should be marked private, if this is the case.

Reply:
i removed it, because it only was a forward to layoutPages(w,h);